fastestimator/op/numpyop/univariate/brightness.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2021 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import random
+from typing import Any, Dict, Iterable, List, Union
+
+import numpy as np
+from PIL import Image, ImageEnhance
+
+from fastestimator.op.numpyop.numpyop import NumpyOp
+from fastestimator.util.traceability_util import traceable
+from fastestimator.util.util import param_to_range
+
+
+@traceable()
+class Brightness(NumpyOp):
+    """Randomly change the brightness of an image.
+
+    Args:
+        inputs: Key(s) of images to be modified.
+        outputs: Key(s) into which to write the modified images.
+        mode: What mode(s) to execute this Op in. For example, "train", "eval", "test", or "infer". To execute
+            regardless of mode, pass None. To execute in all modes except for a particular one, you can pass an argument
+            like "!infer" or "!train".
+        limit: Factor range for changing brightness.
```

Comment:
Can we state the min/max bounds for the arg here? is it -1 to 1?

Reply:
There isn't any bound like that. I will add guidance for the factor like: the factor of 0.0 gives a black image. A factor of 1.0 gives the original image.